src/studio/src/designer/frontend/shared/components/molecules/AltinnPopoverSimple.tsx

```diff
@@ -131,7 +131,7 @@ const AltinnPopoverComponent = (props: IAltinnPopoverProps) => {
                 id={props.btnSecondaryId}
                 color='primary'
                 className={classes.buttonCancel}
-                onClick={props.handleClose}
+                onClick={() => props.handleClose('close')}
```

Comment:
arrow function in render could have performance issues, better to create a function inside the component that handles this?

Reply:
Refactored.